### ISSUE TITLE
Accessibility: Conduct a11y review of Spotlight core

### ISSUE THREAD

**Issue by caaster** [projectblacklight/spotlight#3530]
### Issue Description - WCAG standard
Although institutions such as Stanford have conducted an a11y review of their local Spotlight instance, to date there has not been an a11y review of Spotlight core. Conducting a review will help us be more confident about our a11y work to date, and pinpoint any areas that still require a11y work.

### Method of discovery
Siteimprove, Wave, keyboard navigation, VoiceOver etc.

**Comment by taylor-steve** [comment]
The group believes we mostly ran through a scan last year. I wonder if we can use this issue to either [add additional axe specs](https://github.com/projectblacklight/spotlight/issues/3282) or otherwise routinize the Spotlight a11y testing process.

**Comment by JesiBuell79** [comment]
As this seems to be done, can we close?

**Comment by corylown** [comment]
We have #3282 to capture adding axe specs. Closing this one.
